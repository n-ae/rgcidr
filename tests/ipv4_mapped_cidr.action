::ffff:0:0/96 tests/ipv4_mapped_ipv6.given
