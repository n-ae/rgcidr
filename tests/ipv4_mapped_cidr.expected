::ffff:192.168.1.1
::ffff:10.0.0.1
::ffff:172.16.0.1
::ffff:127.0.0.1
::ffff:8.8.8.8
